pg_search/src/lib.rs

```diff
@@ -13,6 +13,9 @@ extension_sql_file!("../sql/_bootstrap_quickstart.sql");
 #[pg_guard]
 pub unsafe extern "C" fn _PG_init() {
     telemetry::posthog::init("pg_search Deployment");
+
+    // Initalize Grand Unified Configuration (GUC) variables.
+    shared::gucs::init();
```

Comment:
love that haha, is the idea that we will have configuration variables beyond logging that would get added to this down the line?

Reply:
This actually isn't my concept at all! It's what Postgres calls its configuration parameters:
https://pgpedia.info/g/guc.html

Now that I've finalized this PR, I've decided to encapsulate this initialization inside a `ParadeLogsGlobal` struct, which is what initialize instead now at start time. This will be a more consistent interface for future us in our extensions.